x/staking/keeper/proxy.go

```diff
@@ -5,7 +5,7 @@ import (
 	"github.com/okex/okchain/x/staking/types"
 )
 
-// ClearProxy clears the ProxyAddress on binded delegators
+// ClearProxy clears the ProxyAddress on the delegators who has bound
```

Comment:
should change to " // ClearProxy clears the ProxyAddress on the delegator who has bound"

Reply:
Thx!